src/index.html

```diff
@@ -6,13 +6,51 @@
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+          crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
+
     <title>Product cards</title>
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card">
+      <img src="..\src\images\imac.jpeg" alt="Screen photo"
+          class="card__picture">
```

Comment:
Keep your attributes correctly formatted
If the HTML-element has long attribute values or number of attributes is more than 2 - start each one, including the first, on the new line with 2-space indentation related to tag. Tag’s closing bracket should be on the same level as opening one.

```
<input
  type="text" 
  name="surname" 
  id="surname"
  required
>
```